src/backend/src/routes/projects.routes.ts

```diff
@@ -9,18 +9,8 @@ const projectRouter = express.Router();
 projectRouter.get('/', ProjectsController.getAllProjects);
 projectRouter.get('/link-types', ProjectsController.getAllLinkTypes);
 projectRouter.get('/:wbsNum', ProjectsController.getSingleProject);
-projectRouter.post(
-  '/create',
-  intMinZero(body('crId')),
-  nonEmptyString(body('name')),
-  intMinZero(body('carNumber')),
-  nonEmptyString(body('summary')),
-  validateInputs,
-  ProjectsController.createProject
-);
-projectRouter.post(
-  '/edit',
-  intMinZero(body('projectId')),
+
+const projectValidators = [
```

Comment:
I created a list of common validators which are used in both endpoints. If required, I can convert it into a function that adds the optional fields when required